code/modules/particles/byond_particles/emitter/smoke.dm

```diff
@@ -0,0 +1,33 @@
+/obj/emitter/fire_smoke
+	alpha = 150
+	particles = new/particles/fire_smoke
+
+/obj/emitter/fire_smoke/Initialize(mapload)
+	. = ..()
+	add_filter("blur", 1, list(type="blur", size=3))
```

Comment:
What is ``1`` here? Can you use named args?

Reply:
tried add_filter("blur", priority=1 but throws an error for whatever reason

Reply:
``add_filter(name = "blur", priority = 1, params = list(type="blur", size=3))``

I do not know what the error is so I have to assume you did not put params as well. If you are doing named args then all args after the first named one, also must be named.